### DISCUSSION TITLE
.98 version

### DISCUSSION THREAD

**Discussion by TheoBromine2022** [jon-harper/OmniBox#29] [General]
Heyo. you said you were pretty close to a .98 version. Are there any differences in the base between the current version and .98? I had to cannibalize my e5+ to get my e5pro working, not to mention the stock e5+ doesnt have estep calibration so I'm, for now, limited to the e5pro to print the enclosure for my e5+. so I wont be using the modified e5+ monolithic base I made, but rather im using the standard base to get the build started. Just wanting to know before I get started if there would be any incompatibilities down the line.

**Comment by jon-harper** [comment]
I wound up releasing 0.9.8 without making a Reddit announcement. I visually overhauled the documentation and thought it would be ready pretty quickly after that. I also have your lid idea implementations done and wanted to get those in. Instead I've been under the weather and won't have the release out until next week, likely.

Basically, I don't want to spam the Reddit 3D printing subs with release announcements a week apart. I thought they *would* be a week apart, so I didn't announce 0.9.8 on any subs. That backfired, and I apologize if you've been waiting on me. 🙁

------

Anyway, rather than ask which version you printed, it boils down to if the base has an IEC power socket built in to the back. If it does have one, you're good.

Top right on the pic here:
 
![base_wiring](https://user-images.githubusercontent.com/11852387/189508862-dd1a91a2-0807-4265-9914-bb95e39e3862.jpg)

**Reply by TheoBromine2022** [reply]
I havent printed anything, I just got my ender 5 pro working a few hours ago, so no waiting here either. youre good! Since its a new build I'll go ahead and get started tomorrow! get better!

**Reply by jon-harper** [reply]
Oh, right! The new documentation words it better, but [this info may help](https://jon-harper.github.io/OmniBox/#version-stability-guarantee). In short: the project is stable enough that parts are compatible between versions now and moving forward.

**Reply by TheoBromine2022** [reply]
I will also add that for your documentation/bill of materials it wouldnt hurt to include spade disconnects and what gauge wire you are using. I cant tell from your photo but it looks like those disconnects are exposed which definitely gives me some reservations.

**Reply by jon-harper** [reply]
Good point about wiring. I've built so many of these it's easy to overlook some of the stuff I reuse. The wiring I'm using is 16awg.

**Comment by jon-harper** [comment]
Hey, not sure if you've tried this, but you can send the M92 command via USB/serial and change the steps-per-mm, e.g. `M92 E93.0` sets it to 93mm. Follow with `M500` to save to EEPROM.

**Reply by TheoBromine2022** [reply]
Ah, no rush on fixing that. I already have an octopus on hand and I cannibalized the hotend from the 5+ at the moment anyway.

**Comment by TheoBromine2022** [comment]
![20220912_212403](https://user-images.githubusercontent.com/111667850/189910184-a5d559e5-688a-413a-bdcb-39200ffd3b0e.jpg)


64 hour print in ultra quality for the fun of it.

**Reply by jon-harper** [reply]
Quick question: this is the front base? Do you have tree supports enabled? I'm trying to figure out if this is a slicer setting I should mention in the assembly guide, possibly the support overhang angle.

**Reply by TheoBromine2022** [reply]
Yeah you can see the start of the tree in the upper right corner. I'm also printing it with the top facing the build plate to minimize the bridging on the overlaps

**Reply by jon-harper** [reply]
Gotcha. That's the power switch supports, not the fan! I'll make a note about support overhang angle in the docs anyway, as the front main body has some steep angles.

Adaptive layer height is very handy with the main body, by the way. 0.2mm with 0.08 maximum deviation does amazing stuff to give great overhangs without supports.

**Reply by TheoBromine2022** [reply]
PETG bridging is still something im struggling with and I think that power switch slot would look sloppy at any layer height at the moment for me. as for tree supports I will also add that I had to manually override cura for 90% of the trees it wanted to place. since i know the power switch is there I also had no qualms putting a tree support inside the model as well.

**Reply by TheoBromine2022** [reply]
![16631555039887840519405199984818](https://user-images.githubusercontent.com/111667850/190144030-4082856d-392a-49cf-a4a8-cd2021c9f854.jpg)
